build.gradle

```diff
@@ -10,6 +10,13 @@ archivesBaseName = project.archives_base_name
 version = project.mod_version
 group = project.maven_group
 
+repositories {
+    // Add repositories to retrieve artifacts from in here.
+    // Loom adds the Fabric maven automatically so you don't
+    // have to. See https://docs.gradle.org/current/userguide/declaring_repositories.html
+    // for more information about repositories.
```

Comment:
```suggestion
    // Add repositories to retrieve artifacts from in here.
    // You should only use this when depending on other mods because
    // Loom adds the essential maven repositories to download Minecraft and libraries from automatically.
    // See https://docs.gradle.org/current/userguide/declaring_repositories.html
    // for more information about repositories.
```